packages/cli/README.md

```diff
@@ -39,13 +39,13 @@ glaze COMMAND
 - [`glaze pin:add STREAMID`](#glaze-pinadd-streamid)
 - [`glaze pin:ls [STREAMID]`](#glaze-pinls-streamid)
 - [`glaze pin:rm STREAMID`](#glaze-pinrm-streamid)
-- [`glaze stream:commits STREAMID`](#glaze-streamcommits-streamid)
-- [`glaze stream:state STREAMID`](#glaze-streamstate-streamid)
-- [`glaze tile:create CONTENT`](#glaze-tilecreate-content)
-- [`glaze tile:deterministic METADATA`](#glaze-tiledeterministic-metadata)
-- [`glaze tile:content STREAMID`](#glaze-tileshow-streamid)
-- [`glaze tile:update STREAMID CONTENT`](#glaze-tileupdate-streamid-content)
-- [`glaze tile:watch STREAMID`](#glaze-tilewatch-streamid)
+- [`glaze stream:commits STREAMID SYNC_OPTION`](#glaze-streamcommits-streamid)
+- [`glaze stream:state STREAMID SYNC_OPTION`](#glaze-streamstate-streamid)
+- [`glaze tile:create CONTENT SYNC_OPTION`](#glaze-tilecreate-content)
```

Comment:
create doesn't need to expose a sync option, it should only be set on query-like commands, not write commands